ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/GhcrIntegrationForm.tsx

```diff
@@ -273,6 +273,7 @@ function GhcrIntegrationForm({
                         fieldId="config.skipTestIntegration"
                         touched={touched}
                         errors={errors}
+                        helperText="Enable this option for anonymous access with empty username and password"
```

Comment:
@stehessel One question whether to replace **password** with **token** according to discussion:
https://github.com/stackrox/stackrox/pull/13427/files#r1871127297

Reply:
Good point, will do.